packages/server/src/definitions/openapi.ts

```diff
@@ -0,0 +1,964 @@
+        };
+      };
+    };
+  };
+}
+
+export interface components {
+  schemas: {
+    application: {
+      /** @description The name of the app. */
+      name: string;
+      /** @description The URL by which the app is accessed, this must be URL encoded. */
+      url: string;
+    };
+    applicationOutput: {
+      data: {
+        /** @description The name of the app. */
+        name: string;
+        /** @description The URL by which the app is accessed, this must be URL encoded. */
+        url: string;
+        /** @description The ID of the app. */
+        _id: string;
+        /**
+         * @description The status of the app, stating it if is the development or published version.
+         * @enum {string}
+         */
+        status: "development" | "published";
+        /** @description States when the app was created, will be constant. Stored in ISO format. */
+        createdAt: string;
+        /** @description States the last time the app was updated - stored in ISO format. */
+        updatedAt: string;
+        /** @description States the version of the Budibase client this app is currently based on. */
+        version: string;
+        /** @description In a multi-tenant environment this will state the tenant this app is within. */
+        tenantId?: string;
+        /** @description The user this app is currently being built by. */
+        lockedBy?: { [key: string]: unknown };
+      };
+    };
+    /** @description The row to be created/updated, based on the table schema. */
+    row: {
+      [key: string]:
+        | string
+        | { [key: string]: unknown }
+        | number
+        | unknown[]
+        | boolean;
+    };
+    searchOutput: {
+      /** @description An array of rows, these will each contain an _id field which can be used to update or delete them. */
+      data: { [key: string]: unknown }[];
+      /** @description If pagination in use, this should be provided. */
+      bookmark?: string | number;
+      /** @description If pagination in use, this will determine if there is another page to fetch. */
+      hasNextPage?: boolean;
+    };
+    rowOutput: {
+      /** @description The row to be created/updated, based on the table schema. */
+      data: {
+        /** @description The ID of the row. */
+        _id: string;
+        /** @description The ID of the table this row comes from. */
+        tableId: string;
+      } & {
+        [key: string]:
+          | string
+          | { [key: string]: unknown }
+          | number
+          | unknown[]
+          | boolean;
+      };
+    };
+    /** @description The table to be created/updated. */
+    table: {
+      /** @description The name of the table. */
+      name: string;
+      /** @description The name of the column which should be used in relationship tags when relating to this table. */
+      primaryDisplay?: string;
+      schema: {
+        [key: string]:
+          | {
```

Comment:
It would be nice if we could get these `|` blocks as named types, atm it's a bit unclear what each one represents

Reply:
I've removed the type definition, apparently in OpenAPI the defined way to say that its any type is with no type!